packages/SwingSet/test/metering/metered-code.js

```diff
@@ -22,6 +25,10 @@ export function meterMe(log2, explode = 'no') {
     } else if (explode === 'allocate') {
       // eslint-disable-next-line no-unused-vars
       const universe = Array(4e9); // more like 1e24, really
+      let big = '1234';
```

Comment:
Oh, I like this even more than I realized.. by starting with the `Array(4e9)`, the Node.js version will fail quickly and quietly, and it won't even attempt the more-expensive actual allocation. Only XS will do something large that might thrash the system and risk the wrath of the OOM Killer.